### ISSUE TITLE
Incomplete Documentation

### ISSUE THREAD

**Issue by MahmudHAMID** [TanStack/table#5739]
### TanStack Table version

v8.20.5

### Framework/Library version

core

### Describe the bug and the steps to reproduce it

The following docs are incomplete:

1. docs/guide/expanding.md
2. docs/guide/fuzzy-filtering.md
3. docs/guide/global-faceting.md
4. docs/guide/global-filtering.md
5. docs/guide/grouping.md
6. docs/guide/row-pinning.md
7. docs/guide/virtualization.md

### Your Minimal, Reproducible Example - (Sandbox Highly Recommended)

N/A

### Do you intend to try to help solve this bug with your own PR?

None

### Terms & Code of Conduct

- [x] I agree to follow this project's Code of Conduct
- [x] I understand that if my bug cannot be reliable reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
Volunteering to help fill these in?

**Comment by mahmudhamid** [comment]
> Volunteering to help fill these in?

I'll see what I can do.

**Comment by KevinVandy** [comment]
> Also the examples outside of React are limited.
> 
> For example the row expansion is a common task but an example only exists for React.
> 
> I can help with the Angular examples if needed. 

Yeah, of course. Most non-react examples are community contributions. We need more framework specific TanStack champions.
